strapi/strapi
# fix: uid availability check

### What does it do?

Fixes about the UID field availibility check:
- the check should not run if you haven't changed the value of the uid field
- the availability cache should be cleared when you save or update a content type

Also fixes the initialValue from the useField hook which was broken

### How to test it?

- Create a content type with a UID field
- Save an entry with a value in the UID field
- You should no longer see "available" or "unavailable" when nothing was changed
- Create a new entry in the same content type
- If you enter the same UID as the first entry, it should say unavailable. Anything else should be available

### Related issue(s)/PR(s)

fixes [#20598](https://github.com/strapi/strapi/issues/20598)
